jans-auth-server/common/src/main/java/io/jans/as/common/util/RedirectUri.java

```diff
@@ -67,12 +69,14 @@
     private String nestedSharedSecret;
     private JSONObject nestedJsonWebKeys;
     private AbstractCryptoProvider cryptoProvider;
+    private Jwe jwtForEncrypt;
```

Comment:
We have to strive to keep variables as narrow in scope as possible. This one went to class level while it has to be local to method without obvious reason. In short, lets drop it.